src/Operations.jl

```diff
@@ -1405,6 +1406,10 @@ function sandbox(fn::Function, ctx::Context, target::PackageSpec, target_path::S
             cp(sandbox_project, tmp_project)
             chmod(tmp_project, 0o600)
         end
+        if force_latest_compat
+            Types.force_latest_compat(tmp_project)
```

Comment:
Why reparse the project and resave it here, can't you  tweak the compat before it is written?

Reply:
I'm not sure I follow. There are two possible places that the project could come from:
1. `Types.write_project(sandbox_project_override, tmp_project)` - line 1404
2. `cp(sandbox_project, tmp_project)` - line 1406

For case 1, I'd have to modify the `Types.write_project` function and do the compat-tweaking there. And then for case 2, I'd have to reparse and resave the project anyway, right?

So it seemed simpler to just have the compat-tweaking in a single place.

Maybe I'm misunderstanding what you're suggesting?